File: java/Hardware.java
public class Hardware {
    
    private String brand;
    private String model;

    Hardware(){
    }

    //get dan set hardware
    
    public void setBrand(String brand){
        this.brand = brand;
    }

    public String getBrand(){
        return this.brand;
    }

    public void setModel(String model){
        this.model = model;
    }

    public String getModel(){
        return this.model;
    }

}

File: c++/Hardware.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string.h>

using namespace std;

    class Hardware
    {
        private:
            // atribut
            string brand;
            string model;

        public:

        //konstruktor
        Hardware(){
        }

        //get dan set Hardware
        
        void setBrand(string brand){
            this->brand = brand;
        }

        string getBrand(){
            return this->brand;
        }

        void setModel(string model){
            this->model = model;
        }

        string getModel(){
            return this->model;
        }

        //destructor
        ~Hardware(){
        }


    };